File: BOJ/구현/2490.py
# 2490 윷놀이
# https://www.acmicpc.net/problem/2490
for i in range(3):
    li=list(map(int,input().split()))
    f = b = 0
    for j in range(len(li)):
        if li[j]==1:
            b+=1
        else:
            f+=1
    if f==1 and b==3:
        print('A')
    elif f==b==2:
        print('B')
    elif f==3 and b==1:
        print('C')
    elif f==4:
        print('D')
    else:
        print('E')

File: BOJ/구현/2490.cpp
// 2490 윷놀이
// https://www.acmicpc.net/problem/2490
#include <iostream>

using namespace std;

int main(void) {
    int arr[4];

    for (int i = 0; i < 3; ++i)
    {
        for (int j = 0; j < 4; ++j)
        {
            cin >> arr[j];
        }

        int front = 0, back = 0;

        for (int j = 0; j < 4; ++j)
        {
            if (arr[j])
            {
                front++;
            } else {
                back++;
            }
        }

        if (back == 1 && front == 3) cout << 'A' << '\n';
        else if (back == 2 && front == 2) cout << 'B' << '\n';
        else if (back == 3 && front == 1) cout << 'C' << '\n';
        else if (back == 4 && front == 0) cout << 'D' << '\n';
        else cout << 'E' << '\n';
    }
    return 0;
}